### PR TITLE
fix settings avform

### LINKED ISSUE #3478: The speaker audio control periodically set to minimum without any user action
Debian 32bit, qTox updated yesterday
Reproducible: Always. Just let it run for some time.

### LINKED ISSUE #3480: Voice quality dropping when sending files
OS: Linux: Fedora 24
qTox version: v1.4.0-240-ga01293f-obs
Commit hash: a01293f81e58409436f7bab897cc6c2fa2a66f63
toxcore: 0.0.0
Qt: 5.6.0

Reproducible: Always
1. Call a friend
2. Send a file to that friend (doesn't matter what file)
3. Listen

Observed Behavior:
When speaking to a friend while sending a file to him at the same time the voice quality drops a lot.

Expected Behavior:
When speaking to a friend while sending a file to him at the same time the voice quality to stay the same or at least to be good enough, so that we can speak to each other.

### PR DESCRIPTION
**Pull request by antis81**
Fix some things in AVForm:
- [x] initialize audio output volume from settings (fixes #3478)
- [x] rename ui file and class -> it integrates better with the code, e.g. allows switch between ui/code etc.
- [x] fix type conversions and comparisons concerning video resolution and frame rate

### CODE REVIEW

**Comment by zetok** [comment]
> initialize audio output volume from settings -> requires some additional work (fixes #3480)

Not #3478 ?

**Comment by antis81** [comment]
Erm, yep. Referenced the wrong PR by accident. It is going to fix #3478.Corrected in description!

File: src/widget/form/settings/avform.h [hunk 1]
```diff
@@ -61,7 +60,7 @@ private slots:
     // audio
     void onAudioInDevChanged(int deviceIndex);
     void onAudioOutDevChanged(int deviceIndex);
-    void onPlaybackValueChanged(int value);
+    void on_playbackSlider_valueChanged(int value);
```
Comment (sudden6) [inline src/widget/form/settings/avform.h:62]: this line doesn't quite match our naming conventions.

Comment (antis81) [inline src/widget/form/settings/avform.h:62]: It's an autoconnection, it has to be this way. Others will change the same.

**Comment by antis81** [comment]
@sudden6 Ready for merge.

**Comment by zetok** [comment]
on 4e56494d46ac2e72e9b072924c5196c0d9fcb2c2

No idea whether related to the PR; buttons are green and don't indicate that there's an ongoing call, not sure what caused this, I've went into settings & rescanned devices & set newly connected audio device as input.
buttons

Note that I can't toggle muting microphone / speakers when buttons don't indicate that there's an ongoing call.

Also, after switching to the new audio device there's no longer any voice in the call – voice is (transmitted?) only when I have A/V settings tab open, otherwise there's only silence; according to `pavucontrol` qTox doesn't even record audio during the call(!).

**Comment by antis81** [comment]
> No idea whether related to the PR; buttons are green and don't indicate that there's an ongoing call, not sure what caused this, I've went into settings & rescanned devices & set newly connected audio device as input.

The observed behaviour is completely unrelated to this PR.

**Comment by zetok** [comment]
> The observed behaviour is completely unrelated to this PR.

yep, #3521

**Pull request merged**